### PR TITLE
Add Fleet info to Rancher docs

### PR DESCRIPTION
**Pull request by catherineluse**

### CODE REVIEW

File: content/rancher/v2.x/en/deploy-across-clusters/fleet/_index.md [hunk 1]
```diff
@@ -1,8 +1,28 @@
...
+Fleet can manage deployments from git of raw Kubernetes YAML, Helm charts, or Kustomize or any combination of the three. Regardless of the source, all resources are dynamically turned into Helm charts, and Helm is used as the engine to
+deploy everything in the cluster. This give a high degree of control, consistency, and auditability. Fleet focuses not only on the ability to scale, but to give one a high degree of control and visibility to exactly what is installed on the cluster.
+
+### Accessing Fleet in the Rancher UI
+
+Fleet comes preinstalled in Rancher v2.5. To access it, go to the **Cluster Explorer** in the Rancher UI. In the top left dropdown menu, click **Cluster Explorer > Fleet.** On this page, you can edit Kubernetes resources and cluster groups managed by Fleet.
+
+### GitHub Repository
+
+The Fleet Helm charts are available [here.](https://github.com/rancher/fleet)
```
Comment (StrongMonkey) [inline content/rancher/v2.x/en/deploy-across-clusters/fleet/_index.md:24]: Link should be pointed to the latest release. https://github.com/rancher/fleet/releases/latest

File: content/rancher/v2.x/en/deploy-across-clusters/fleet/_index.md [hunk 2]
```diff
@@ -1,8 +1,28 @@
...
+title: Fleet - GitOps at Scale
+shortTitle: Fleet
 weight: 1
 ---
 
 _Available as of Rancher v2.5_
 
-This section is under construction.
\ No newline at end of file
+Fleet is GitOps at scale. Fleet is designed to manage up to a million clusters. It's also lightweight enough that is works great for a [single cluster](https://fleet.rancher.io/single-cluster-install/) too, but it really shines when you get to a large scale. By large scale we mean either a lot of clusters, a lot of deployments, or a lot of teams in a single organization.
```
Comment (StrongMonkey) [inline content/rancher/v2.x/en/deploy-across-clusters/fleet/_index.md:9]: Should `large scale` be linked to https://fleet.rancher.io/multi-cluster-install/?

File: content/rancher/v2.x/en/deploy-across-clusters/fleet/_index.md (hunk 1 above)
Comment (catherineluse) [reply content/rancher/v2.x/en/deploy-across-clusters/fleet/_index.md:24]: I've added the link

**Comment by catherineluse** [comment]
I made the changes here https://github.com/rancher/docs/commit/408a630b5c36a5292f15ffb2d6795f73ead28009

**Pull request merged**